src/main/resources/application.properties

```diff
@@ -1,10 +1,8 @@
-spring.jpa.show-sql=true
-
-spring.datasource.url=jdbc:mysql://localhost:3306/pmtool?useUnicode=true&useJDBCCompliantTimezoneShift=true&useLegacyDatetimeCode=false&serverTimezone=UTC
-spring..datasource.username = molnarsandor
-spring..datasource.password = password
-
+#Database
 spring.jpa.database-platform=org.hibernate.dialect.MySQL5Dialect
+spring.jpa.hibernate.ddl-auto=update
```

Comment:
Hobbi projektekben ahogy fentebb is írtam nagyon jó, de egy Flyway vagy Liquibase megnézése és a projektbe építése szerintem megfontolandó.